en/flight_controller/ocpoc_zynq.md

```diff
@@ -47,3 +47,16 @@ make aerotenna_ocpoc_ubuntu
 
 See the user hub page [PX4 on OcPoC Zynq Mini](https://aerotenna.readme.io/docs/px4-setup) for full step-by-step instructions for first time setup of PX4 on OcPoC-Zynq. 
 Note: At the present time PX4 firmware must be built from source as no pre-built binaries for OcPoC-Zynq are supplied through QGroundControl.
+
+## Serial Port Mapping
+
+OcPoc Port | Device | Port
```

Comment:
This one a bit different but I assume still useful? The information is pretty much just from here: https://github.com/PX4/Firmware/blob/master/boards/aerotenna/ocpoc/default.cmake#L13